pkg/state/state.go

```diff
@@ -2967,6 +2972,19 @@ func (st *HelmState) flagsForDiff(helm helmexec.Interface, release *ReleaseSpec,
 		flags = st.appendSuppressOutputLineRegexFlags(flags, release, suppressOutputLineRegex)
 	}
 
+	if opt.TakeOwnership || st.HelmDefaults.TakeOwnership || release.TakeOwnership {
```

Comment:
```suggestion
	if opt.TakeOwnership || release.TakeOwnership || st.HelmDefaults.TakeOwnership {
```
`HelmDefaults.TakeOwnership` should act as fallback

Reply:
@hubertbits thanks so much.